### ISSUE TITLE
Update to Debian Stretch

### ISSUE THREAD

**Issue by mlowicki** [getsentry/docker-sentry#140]
Debian 8 (Jessie) is now an obsolete stable release (https://www.debian.org/releases/). 

Debian Jessie will be in LTS phase since June 2018 - "Debian LTS will not be handled by the Debian security team, but by a separate group of volunteers and companies interested in making it a success.".

Packages in obsolete releases do not receive patches for all security vulnerabilities like:
* https://security-tracker.debian.org/tracker/source-package/openssh
* https://security-tracker.debian.org/tracker/source-package/curl
* https://security-tracker.debian.org/tracker/source-package/binutils

**Comment by mattrobenolt** [comment]
Yup, I will after we test and fix our stuff to work on Stretch first.

**Comment by J0WI** [comment]
How about Alpine?
https://docs.docker.com/samples/library/python/#pythonversion-alpine

**Comment by mattrobenolt** [comment]
Sentry does not run on Alpine.

**Comment by J0WI** [comment]
What exactly does not work on Alpine? I assume Python applications are not that hard to port. Would you accept a PR?

**Comment by mattrobenolt** [comment]
Most of our dependencies that are compiled in C or Rust. Alpine is not trivially compatible unless you’re pure python. 

I assure you, I’ve made an attempt before.

**Comment by NullP0interEx** [comment]
Project can not be built anymore: 

`Get:1 http://security.debian.org jessie/updates InRelease [44.9 kB]
Get:2 http://security.debian.org jessie/updates/main amd64 Packages [824 kB]
Ign http://deb.debian.org jessie InRelease
Ign http://deb.debian.org jessie-updates InRelease
Get:3 http://deb.debian.org jessie Release.gpg [2420 B]
Ign http://deb.debian.org jessie-updates Release.gpg
Get:4 http://deb.debian.org jessie Release [148 kB]
Ign http://deb.debian.org jessie-updates Release
Get:5 http://deb.debian.org jessie/main amd64 Packages [9098 kB]
Err http://deb.debian.org jessie-updates/main amd64 Packages
  
Err http://deb.debian.org jessie-updates/main amd64 Packages
  
Err http://deb.debian.org jessie-updates/main amd64 Packages
  
Err http://deb.debian.org jessie-updates/main amd64 Packages
  
Err http://deb.debian.org jessie-updates/main amd64 Packages
  404  Not Found
Fetched 10.1 MB in 10s (1011 kB/s)
W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/main/binary-amd64/Packages  404  Not Found

E: Some index files failed to download. They have been ignored, or old ones used instead.`

**Comment by mattrobenolt** [comment]
https://github.com/docker-library/official-images/pull/5609

I'm not sure if there's anything I can do to force this to be fixed, because the base images are fixed. So it _can_ build.

But with that said, we have been using a build based on stretch ourselves in a test environment without any issues, so we'll probably just move forward with that.

**Comment by NullP0interEx** [comment]
can you fix that in the master?

**Comment by mattrobenolt** [comment]
I assume the world will fix itself soon. There's nothing I can really do, since it's up to the base images being rebuilt. I can shove in a hack or so, but this isn't really a problem for now.

**Comment by yosifkit** [comment]
It would, but we've already moved on from the python that you are `FROM` so sentry won't be rebuilt (#116):

https://github.com/getsentry/docker-sentry/blob/af9e34c918c306e927f7506ea8d22cf57b95d65f/9.0/Dockerfile#L1

https://github.com/docker-library/python/blob/67e7d70195c46f16116a57307c364167cb1302f6/2.7/jessie/slim/Dockerfile#L25

**Comment by mattrobenolt** [comment]
This has been done in our 9.1.0 release. Also bumped to 2.7.16.

**Comment by caseyduquettesc** [comment]
@NullP0interEx you can fix 8.22 and 9.0 with this

```
FROM sentry:8.22-onbuild

RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
RUN sed -i '/deb http:\/\/deb.debian.org\/debian jessie-updates main/d' /etc/apt/sources.list
RUN apt-get -o Acquire::Check-Valid-Until=false update
```
